system/Database/Database.php

```diff
@@ -41,7 +41,7 @@ class Database
 	 * @return   mixed
 	 * @internal param bool $useBuilder
 	 */
-	public function load(array $params = [], string $alias)
```

Comment:
This (and the majority of these changes) are to correct the improper syntax of a required variable following an optimal variable, a big "no no" in PHP 8.

Reply:
They've actually started enforcing that at the language level? Wow. I need to find time to read up on PHP8 changes very soon!

Reply:
It's a depreciation notice:
```
PHP Deprecated:  Required parameter $from follows optional parameter $verbs in /home/runner/work/CodeIgniter4/CodeIgniter4/system/Router/RouteCollection.php on line 991
9
```